src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -12907,7 +12917,7 @@ bool Unit::SetHover(bool enable, bool updateAnimTier /*= true*/)
         SendMessageToSet(packet.Write(), true);
     }
 
-    if (IsCreature() && updateAnimTier && IsAlive() && !HasUnitState(UNIT_STATE_ROOT) && !ToCreature()->IsTemplateRooted())
+    if (IsCreature() && updateAnimTier && IsAlive() && !HasUnitState(UNIT_STATE_ROOT) && !ToCreature()->IsSessile())
```

Comment:
Remove `IsSessile` check here, UNIT_STATE_ROOT already covers that (it is impossible to lose UNIT_STATE_ROOT when IsSessile is true)